### PR TITLE
Finish documenting `bevy_gilrs`

### PR DESCRIPTION
**Pull request by Kanabenki**
# Objective

Finish documenting `bevy_gilrs`.

## Solution

Document the remaining items, add a `warn(missing_docs)` attribute for the crate.

### CODE REVIEW

**Comment by Kanabenki** [comment]
I linked to the gitlab repo of GilRs, would the crates.io or the docs.rs page be preferable ?

**Comment by mockersf** [comment]
I think linking to docs.rs makes more sense, as this will be the place where most people will read it

**Review by JMS55 — APPROVED** [review]

File: crates/bevy_gilrs/src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,10 @@
+//! Systems and type definitions for gamepad handling in Bevy.
+//!
+//! This crate is built on top of [GilRs](https://docs.rs/gilrs/latest/gilrs), a library
```
Comment (killercup) [inline crates/bevy_gilrs/src/lib.rs:3]: I wonder if using `[GilRs](gilrs)` would cause docs.rs to link to the right version?

Comment (Kanabenki) [reply crates/bevy_gilrs/src/lib.rs:3]: Seems like this works properly locally (if the dep doc is built of course), and on docs.rs! ([example I found](https://docs.rs/test8981/0.1.0/test8981/))

**Review by killercup — APPROVED** [review]

**Review by mockersf — APPROVED** [review]

**Pull request merged**